src/main/java/uk/gov/companieshouse/insolvency/data/service/InsolvencyServiceImpl.java

```diff
@@ -118,6 +91,11 @@ public CompanyInsolvency retrieveCompanyInsolvency(String companyNumber) {
                 () -> new DocumentNotFoundException(String.format(
                         "Resource not found for company number: %s", companyNumber)));
 
+        CompanyInsolvency companyInsolvency = insolvencyDocument.getCompanyInsolvency();
```

Comment:
This seems like a hack to deal with dodgy data, if it is I'd prefer fixing any instances where status is an empty list in Mongo. Otherwise, we should amend the insert/update code to never persist an empty list.

Reply:
The size of the Mongo collection might be an issue where a datafix query might take a prohibitively long time to run.
Can we get @Kurt to get a count on the number of documents that need to be fixed?